### PR TITLE
Specify how to handle punctuation

### PR DESCRIPTION
**Pull request by adamwight**
Align the requirements with test cases.

Co-authored-by: Glenn Jackman <glenn.jackman@gmail.com>

### CODE REVIEW

File: exercises/acronym/description.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ Techies love their TLA (Three Letter Acronyms)!
...
 
+Punctuation is handled as follows: hyphens are word separators (like whitespace); all other punctuation can be removed from the input.
+
 For example:
 
 |Input|Output|
 |-|-|
 |As Soon As Possible|ASAP|
 |Liquid-crystal display|LCD|
+|Thank George It's Friday!|TGIF|
```
Comment (SaschaMann) [inline exercises/acronym/description.md:17]: The existing examples don't do it either but I think these should also be part of the tests.

**Review by SaschaMann — APPROVED** [review]

File: exercises/acronym/description.md (hunk 1 above)
Comment (adamwight) [reply exercises/acronym/description.md:17]: Or maybe the other way around, and I should just copy over some of the tests which demonstrate ignored punctuation?
```
|Halley's Comet|HC|
|First In, First Out|FIFO|
```

Or is it fine that the concepts are demonstrated although the exact text isn't reflected in tests?

Comment (ErikSchierboom) [reply exercises/acronym/description.md:17]: > Or is it fine that the concepts are demonstrated although the exact text isn't reflected in tests?

Yeah that's totally fine.

**Review by ErikSchierboom — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Great stuff!

**Pull request merged**